openmdao/core/group.py

```diff
@@ -4345,3 +4345,18 @@ def _ordered_comp_name_iter(self):
                 yield from s._ordered_comp_name_iter()
             else:
                 yield s.pathname
+
+    @property
+    def model_options(self):
```

Comment:
I don't see that this property on `Group` is ever used.. is there a use case for it? is it in need of a unit test?